tests/oss-fuzz/build.sh

```diff
@@ -40,13 +40,14 @@ set -eu
 
 # Build dav1d with sanitizer flags.
 # Adds extra flags: -Db_sanitize=$SANITIZER -Db_lundef=false, and -Denable_asm=false for msan
+DAV1D_EXTRA_FLAGS=""
 if [ "$SANITIZER" != "coverage" ] && [ "$SANITIZER" != "introspector" ]
 then
-  export DAV1D_EXTRA_FLAGS="-Db_sanitize=$SANITIZER -Db_lundef=false"
+  DAV1D_EXTRA_FLAGS="-Db_sanitize=$SANITIZER -Db_lundef=false"
```

Comment:
Optional: Since `DAV1D_EXTRA_FLAGS` has been initialized an empty string, we can append to it here.

Reply:
Done